packaging/nnfw.spec

```diff
@@ -125,22 +131,31 @@ NPU daemon for optimal management of NPU hardware
 
 %define install_dir %{_prefix}
 %define install_path %{buildroot}%{install_dir}
-%define build_env NNFW_WORKSPACE=build
-%define build_options -DCMAKE_BUILD_TYPE=%{build_type} -DTARGET_ARCH=%{target_arch} -DTARGET_OS=tizen -DENABLE_TEST=off -DBUILD_MINIMAL_SAMPLE=on
+%define nnfw_workspace build
+%define build_env NNFW_WORKSPACE=%{nnfw_workspace}
 
-# Set option for test build (and coverage test build)
+# Path to install test bin and scripts (test script assumes path Product/out)
+# TODO Share path with release package
 %define test_install_home /opt/usr/nnfw-test
 %define test_install_dir %{test_install_home}/Product/out
 %define test_install_path %{buildroot}%{test_install_dir}
-%define coverage_option %{nil}
+
+# Set option for test build (and coverage test build)
+%define option_test -DENABLE_TEST=OFF
+%define option_coverage %{nil}
 %define test_suite_list infra/scripts tests/scripts
-%define test_build_type %{build_type}
+
+%if %{test_build} == 1
+# ENVVAR_ONERT_CONFIG: Use environment variable for runtime core configuration and debug
+%define option_test -DEANBLE_TEST=ON -DENVVAR_ONERT_CONFIG=ON
```

Comment:
```suggestion
%define option_test -DENABLE_TEST=ON -DENVVAR_ONERT_CONFIG=ON
```

Reply:
@hseok-oh I found that nnfw-test build is done w/o this line patch. Any guess ?

Reply:
```cmake
option(ENABLE_TEST "Build Tests using Google Test" ${GTest_FOUND})
```

If `test_build == 1`, build environment will install gtest, then `GTest_FOUND` will set. So `ENABLE_TEST` become `ON`.